index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Styles Conference</title>
+    <link rel="stylesheet" href="stylesheets/reset.css" />
+    <link rel="stylesheet" href="stylesheets/style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@200&display=swap" rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <main class="main-div">
+      <div class="bg-image">
+        <header class="primary-header container">
+          <nav class="nav primary-nav container">
+            <div>
+              <button type="button" class="btn-apple">
+                <!--Apple-->
+                <img
+                  src="images/apple-logo-png.png"
+                  alt="apple-logo"
+                />&nbsp;</button
+              ><!--
+            --><button type="button" class="btn-apple">Store</button
+              ><!--
+            --><button type="button" class="btn-apple">Mac</button
+              ><!--
+            --><button type="button" class="btn-apple">iPod</button
+              ><!--
+            --><button type="button" class="btn-apple">iPhone</button
+              ><!--
+            --><button type="button" class="btn-apple">iPad</button
+              ><!--6.5
+            --><button type="button" class="btn-apple">iTunes</button
+              ><!--
+            --><button type="button" class="btn-apple">Support</button
+              ><!--
+            --></div>
+            <div>
+              <input type="text" class="in-apple" />
+            </div>
+          </nav>
+        </header>
+        <section class="banner container">
+          <p class="catch-phrase">What will your verse be?</p>
+          <h3>
+            Everyone has something to share. See how people are using <br />
+            iPad in their unique ways to add to the world's story.
+          </h3>
```

Comment:
- [**optional**] Here, I noticed a misuse of semantic HTML elements. The element in the `<p>` tag is considered to be the title of the site and so should have a header tag and which is higher than the header tag `<h3>` you gave to the subtitle. So, you should consider giving it an appropriate semantic tag.